hack/make/.go-autogen.ps1

```diff
@@ -27,63 +44,109 @@ Function Get-BuildDateTime() {
 
-    # Compile the messages
-    windmc hack\make\.resources-windows\event_messages.mc -h autogen\winresources\tmp -r autogen\winresources\tmp
-    if ($LASTEXITCODE -ne 0) { Throw "Failed to compile event message resources" }
+    $mkwinresContents = '{
+  "RT_GROUP_ICON": {
+    "#1": {
+      "0409": "../../winresources/docker.ico"
+    }
+  },
+  "RT_MANIFEST": {
+    "#1": {
+      "0409": {
+        "identity": {},
+        "description": "Docker Engine",
+        "minimum-os": "vista",
+        "execution-level": "",
+        "ui-access": false,
+        "auto-elevate": false,
+        "dpi-awareness": "unaware",
+        "disable-theming": false,
+        "disable-window-filtering": false,
+        "high-resolution-scrolling-aware": false,
+        "ultra-high-resolution-scrolling-aware": false,
+        "long-path-aware": false,
+        "printer-driver-isolation": false,
+        "gdi-scaling": false,
+        "segment-heap": false,
+        "use-common-controls-v6": false
+      }
+    }
+  },
+  "RT_MESSAGETABLE": {
+    "#1": {
+      "0409": "../../winresources/event_messages.bin"
+    }
+  },
+  "RT_VERSION": {
+    "#1": {
+      "0409": {
+        "fixed": {
+          "file_version": "'+(Get-FixQuadVersionNumber($versionMatches.Groups[1].Value))+'.'+(Get-FixQuadVersionNumber($versionMatches.Groups[2].Value))+'.'+(Get-FixQuadVersionNumber($versionMatches.Groups[3].Value))+'.'+(Get-FixQuadVersionNumber($versionMatches.Groups[4].Value))+'",
+          "product_version": "'+(Get-FixQuadVersionNumber($versionMatches.Groups[1].Value))+'.'+(Get-FixQuadVersionNumber($versionMatches.Groups[2].Value))+'.'+(Get-FixQuadVersionNumber($versionMatches.Groups[3].Value))+'.'+(Get-FixQuadVersionNumber($versionMatches.Groups[4].Value))+'",
+          "type": "Unknown"
+        },
+        "info": {
+          "0000": {
+            "CompanyName": "'+$PackagerName+'",
+            "FileVersion": "'+$DockerVersion+'",
+            "LegalCopyright": "Copyright (C) 2015-'+$currentYear+' Docker Inc.",
+            "OriginalFileName": "dockerd.exe",
+            "ProductName": "'+$Product+'",
+            "ProductVersion": "'+$DockerVersion+'",
+            "SpecialBuild": "'+$CommitString+'"
+          }
+        }
+      }
+    }
+  }
+}'
 
-    # If you really want to understand this madness below, search the Internet for powershell variables after verbatim arguments... Needed to get double-quotes passed through to the compiler options.
-    # Generate the .syso files containing all the resources and manifest needed to compile the final docker binaries. Both 32 and 64-bit clients.
-    $env:_ag_dockerVersion=$DockerVersion
-    $env:_ag_gitCommit=$CommitString
+    # Write the file
+    $outputFile="$(Get-Location)\cli\winresources\dockerd\winres.json"
+    if (Test-Path $outputFile) { Remove-Item $outputFile }
+    Set-Content -NoNewline -Path $outputFile -Value $mkwinresContents
```

Comment:
Yes I used `Set-Content` because I needed `-NoNewline` otherwise `goversioninfo` would fail because of the carriage return `\r\n` in the JSON file. Will se with `go-winres`

Reply:
Looks good